onnxruntime/core/optimizer/qdq_transformer/selectors_actions/qdq_selectors.cc

```diff
@@ -547,7 +547,7 @@ bool BatchNormalizationNodeGroupSelector::Check(const GraphViewer& graph_viewer,
                                                 const Node& node,
                                                 const std::vector<const Node*>& dq_nodes,
                                                 const std::vector<const Node*>& q_nodes) const {
-  if (!CheckQDQNodes(graph_viewer, node, dq_nodes, q_nodes)) {
+  if (!CheckQDQNodes(graph_viewer, node, dq_nodes, q_nodes, 3)) {
```

Comment:
I don't know why we did that previously, but QNN doesn't require the mean and var. So it just bring in extra cost if have mean and var quantized.
It will just make it hard for us to maintain if we still want to support mean & var to be quantized.
Also it's safe to ask user re-generate the model and have extra benefit since we are using int32 for bias quantization which is more accurate.